src/main/java/org/mitre/synthea/export/CPCDSExporter.java

```diff
@@ -185,19 +194,26 @@ public static CPCDSExporter getInstance() {
    */
   public void export(Person person, long time) throws IOException {
     String personID = patient(person, time);
+    String payerId = "";
     String type = COVERAGE_TYPES[(int) randomLongWithBounds(0, COVERAGE_TYPES.length - 1)];
     UUID groupId = GROUPIDS[(int) randomLongWithBounds(0, GROUPIDS.length - 1)];
     String groupName = GROUP_NAMES[(int) randomLongWithBounds(0, GROUP_NAMES.length - 1)];
+    String name = PLAN_NAMES[(int) randomLongWithBounds(0, PLAN_NAMES.length -1)];
+    
 
     for (Encounter encounter : person.record.encounters) {
-
       String encounterID = UUID.randomUUID().toString();
-      String payerId = encounter.claim.payer.uuid.toString();
       UUID medRecordNumber = UUID.randomUUID();
       CPCDSAttributes encounterAttributes = new CPCDSAttributes(encounter);
+      if (Boolean.parseBoolean(Config.get("exporter.cpcds.single_payer"))) {
+        payerId = "b1c428d6-4f07-31e0-90f0-68ffa6ff8c76";
+      }
+      else {
+        payerId = encounter.claim.payer.uuid.toString();
+      }
```

Comment:
While interesting, this is unrelated to encoding. Also, why the particular hard-coded UUID?

Reply:
You are correct; many of these changes are unrelated. Old code that snuck in from where I branched.